sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeOperation.cs

```diff
@@ -43,45 +38,48 @@ public class AnalyzeOperation : Operation<AnalyzeOperationResult>
         /// <remarks>
         /// This property can be accessed only after the operation completes successfully (HasValue is true).
         /// </remarks>
-        public override AnalyzeOperationResult Value
-        {
-            get
-            {
-                if (HasCompleted && !HasValue)
-#pragma warning disable CA1065 // Do not raise exceptions in unexpected locations
-                    throw _requestFailedException;
-#pragma warning restore CA1065 // Do not raise exceptions in unexpected locations
-                else
-                {
-                    return OperationHelpers.GetValue(ref _value);
-                }
-            }
-        }
+        public override AsyncPageable<AnalyzeOperationResult> Value => GetValuesAsync();
 
-        /// <summary><c>true</c> if the long-running operation has completed. Otherwise, <c>false</c>.</summary>
+        /// <summary>
+        /// <c>true</c> if the long-running operation has completed. Otherwise, <c>false</c>.
+        /// </summary>
         private bool _hasCompleted;
 
         /// <summary>
         /// Returns true if the long-running operation completed.
         /// </summary>
         public override bool HasCompleted => _hasCompleted;
 
+        /// <summary>
+        /// If the operation has an exception, this property saves its information.
+        /// </summary>
         private RequestFailedException _requestFailedException;
 
-        /// <summary>The last HTTP response received from the server. <c>null</c> until the first response is received.</summary>
+        /// <summary>
+        /// The last HTTP response received from the server. <c>null</c> until the first response is received.
+        /// </summary>
         private Response _response;
 
-        /// <summary>The result of the long-running operation. <c>null</c> until result is received on status update.</summary>
-        private AnalyzeOperationResult _value;
+        /// <summary>
+        /// The result of the long-running operation. <c>null</c> until result is received on status update.
+        /// </summary>
+        private Page<AnalyzeOperationResult> _firstPage;
```

Comment:
Should this comment be `Provides the results for the first page.` like in `AnalyzeHealthcareEntitiesOperation`?

Reply:
yes! thank you!